static/js/settings_notifications.js

```diff
@@ -69,6 +70,17 @@ function change_notification_setting(setting, setting_data, status_element) {
     settings_ui.do_settings_change(channel.patch, '/json/settings/notifications', data, status_element);
 }
 
+function update_desktop_icon_count_display() {
+    var only_show_notifiable = page_params.desktop_icon_count_display === 2;
```

Comment:
Our standard approach is rather than using values like `2`, have a structure like `exports.email_address_visibility_values` that gives these more readable names.  This would improve the code here and especially in the `2 ? 1` below, which is pretty opaque :)

Reply:
Fixed, thanks!